bin/deprecation-detector

```diff
@@ -1,7 +1,12 @@
 #!/usr/bin/env php
 <?php
 
-require __DIR__.'/../vendor/autoload.php';
+if (file_exists(__DIR__ . '/../../../autoload.php')) {
+    // deprecation-detector is part of a composer installation
+    require_once __DIR__ . '/../../../autoload.php';
+} else {
```

Comment:
You could check here if the file being loaded on the next line actually does exist and output a message otherwise (this would be an indicator that you did not run `composer install` before).